Add plugin limitation in terms of users (#207)
mattermost/mattermost-plugin-zoom: docs/README.md

![](.gitbook/assets/42196048-af54d2b8-7e30-11e8-80a0-5e160ae06f03%20%282%29.png)

# Important

> Only Zoom users associated with the Zoom Account that created the Zoom App will be able to use the plugin. You can add users under the Manage Users option in the Zoom Account settings.

Reviewer: ```suggestion
Only Zoom users associated with the Zoom Account that created the Zoom App will be able to use the plugin. You can add these users from the **Manage Users** section in the Zoom Account settings.
```
